### PR TITLE
Fix formatting and wording in intro docs

### PR DESCRIPTION
**Pull request by sake92**
Found some formatting issues with asciidoc, it considered `_` as a start of an italic word.  
It needs to be escaped in some cases, like this: `\_`

Also, I ran some bigger sentences through https://hemingwayapp.com/, and the text was a bit complicated, so I split some of them into 2 simpler ones...

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 1]
```diff
@@ -213,7 +214,7 @@ $ mill -i foo.console              # start a Scala console within your project (
 $ mill -i foo.repl                 # start an Ammonite REPL within your project (in interactive mode: "-i")
 ----
 
-You can run `mill resolve __` to see a full list of the different tasks that are available, `mill resolve foo._` to see
+You can run `mill resolve \__` to see a full list of the different tasks that are available, `mill resolve foo._` to see
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:217]: I think the correct asciidoc approach is to use `+` symbols here.

```suggestion
You can run +mill resolve __+ to see a full list of the different tasks that are available, `mill resolve foo._` to see
```

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 2]
```diff
@@ -514,7 +515,7 @@ Inputs:
 it also displays its source location and a list of input tasks. This is very useful for debugging and interactively
 exploring the structure of your build from the command line.
 
-`inspect` also works with the same `_`/`__` wildcard/query syntaxes that
+`inspect` also works with the same `\_`/`__` wildcard/query syntaxes that
```
Comment (lefou) [inline docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc:518]:
```suggestion
`inspect` also works with the same +_+/+__+ wildcard/query syntaxes that
```

**Review by lefou — COMMENTED** [review]
Thanks for your eagle eyes! I bikeshedded a bit. ;-)

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc (hunk 2 above)
Comment (sake92) [reply docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc]: Fixed

File: docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc [hunk 3]
```diff
@@ -52,8 +53,8 @@ pkg install mill
 
 === Windows
 
-To get started, download Mill from:
-{mill-github-url}/releases/download/{mill-last-tag}/{mill-last-tag}-assembly, and save it as `mill.bat`.
+To get started, download Mill from
+{mill-github-url}/releases/download/{mill-last-tag}/{mill-last-tag}-assembly[here], and save it as `mill.bat`.
```
[the remark this answers is not in the document]
Comment (sake92) [reply docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc]: Fixed

**Review by lefou — APPROVED** [review]
Great, thank you!

**Pull request merged**